src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,202 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = LCDFeatureSet( )
+            obj = obj@FeatureCreators.Base();
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...

```

Comment:
No, they are not. And I'm afraid they are important for identification but destructive for localization, at least the bBinauralRMS set to false.
bNormalizeRMS on the other hand (but with bBinauralRMS set to true) might be helpful for localization too, @ningma97 ? (of course only with retraining of models)

Anyway, since these are preprocessor parameters and thus influence the first processor in the chain, identification and localization AFE processing chains diverge immediately. But at least we could try at the identification side to go down to one ratemap only, leaving us with two in total.